pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -540,46 +548,56 @@ For example, given the module declaration
 
 the evaluation of @racket[(require 'm)] creates the variable @racket[x]
 and installs @racket[10] as its value. This @racket[x] is unrelated to
-any top-level definition of @racket[x].
+any top-level definition of @racket[x]. It's as
+if the variable were actually named ``x in m.''
 
-@;- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+@;------------------------------------------------------------------------
 @subsection[#:tag "module-phase"]{Phases}
 
-A module can be @tech{instantiate}d in multiple @deftech{phases}. A
-phase is an integer that, again, is effectively a prefix on the names
-of module-level definitions. A top-level @racket[require]
+The purpose of @deftech{phases} is to solve the ``eval-when'' problem that
+appears in various Lisp dialects. In particular, phases
+address the necessary separation of names defined at execution time versus
+names defined at compile/macro-expansion time. See the paper
+``Composable and Compilable Macros'' for detailed information @cite["Flatt02"].
+
+A module can be @tech{instantiate}d in multiple @tech{phases}. A
+phase is an integer that, like a module name, is effectively a prefix on the names
+of module-level definitions. Phase 0 is the execution-time phase.
+
+A top-level @racket[require]
 @tech{instantiates} a module at @tech{phase} 0, if the module is not
 already @tech{instantiate}d at phase 0.  A top-level
-@racket[(require (for-syntax ....))] @tech{instantiates} a module at
+@racket[(require (for-syntax ...))] @tech{instantiates} a module at
```

Comment:
I think the 4 dots are intentional to avoid confusion with the `...` transformer.

Reply:
Aha, right. I'll fix that.